content/en/docs/v3.5/op-guide/supported-platform.md

```diff
@@ -47,7 +46,7 @@ tier:
 | Build passes                          | &check;| &check;| &check;|
 | Unit tests pass                       | &check;| &check;|        |
 | Integration and end-to-end tests pass | &check;| &check;|        |
-| Functional and stress tests pass      | &check;|        |        |
+| Robustness and stress tests pass      | &check;|        |        |
```

Comment:
The stress test is also part of the robustness test? :)

Reply:
Updated, thanks 👍🏻